async-query-core/src/test/java/org/opensearch/sql/spark/utils/SQLQueryUtilsTest.java

```diff
@@ -404,15 +407,40 @@ void testAutoRefresh() {
 
   @Test
   void testValidateSparkSqlQuery_ValidQuery() {
```

Comment:
nit: Please use blank line to separate Arrange/Act/Assert section for readability.

Reply:
done